openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -144,13 +144,13 @@ <h2 class="staff-info__title ui-staff__subtitle">
                     </tbody>
                 </table>
             {% elif workflow_status == "waiting" %}
-                <div class="staff-info__final__grade__score">{% trans "The submission is waiting for assessments." %}</div>
+                <div class="staff-info__final__grade__score"><p>{% trans "The submission is waiting for assessments." %}</p></div>
```

Comment:
Added p-tags around these message to make the styling more consistent with other parts of the staff grading UI (a different font was being used without the p-tags).